docs/installation/server-config/config-reference.rst

```diff
@@ -251,6 +252,44 @@ The corresponding environment variable in the
 `KSQL Server image <https://hub.docker.com/r/confluentinc/cp-ksql-server/>`__ is
 ``KSQL_KSQL_FUNCTIONS_SUBSTRING_LEGACY_ARGS``.
 
+.. _ksql_persistence_ensure_value_is_struct:
+
+---------------------------------------
+ksql.persistence.ensure.value.is.struct
```

Comment:
perhaps `ksql.serialization.value.ensure.struct` - that way we can group under `serialization` and `serialization.value`

Reply:
Renamed in recent push...